doc/common-issues.rst

```diff
@@ -0,0 +1,101 @@
+Frequently Encountered Issues
+=============================
+
+.. contents::
+
+Server reports wire version 5, PyMongo requires 6
+-------------------------------------------------
+
+When one attempts to connect to a <=3.4 version server, PyMongo will throw the following error:
+
+.. code-block::
+
+	>>> client.admin.command('ping')
```

Comment:
For all the code examples, indent the lines only 2 spaces like we do for most other code examples.

Reply:
It still looks like these examples are intended one tab instead of 2 spaces.